x-pack/test_serverless/functional/test_suites/security/cypress/tasks/login.ts

```diff
@@ -32,7 +31,7 @@ const sendApiLoginRequest = (
     url: url.toString(),
     body: {
       providerType: 'basic',
-      providerName: isLocalhost(url.hostname) ? 'basic' : 'cloud-basic',
+      providerName: 'basic',
```

Comment:
@jeramysoucy why do we need to make this change? I thought it was intentionally done like that to support tests in ESS?

Reply:
Checked with Larry and got more context around the issue you were trying to solve here - sounds good to me. We can re-consider it in the future if we need to.